cpp/ps0000/problem solve 0.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream>
+#include <iostream> // 2/2
 // اول غلطه كتب @ وهي مفروض #
 // تانى غلطة في كلمه "iostream.h  هي مفروض من غير علامتين تنصيص ومن غير .h
 using namespace std;
 //نسي يكتب الجمله دي
-int main()
+int main() // 1/1
 // كتب كلمه void 
 // كتب Main بدل main 
 {
-  cout << "Hello World \n " ;
+  cout << "Hello, World\n" ; // 3/4
 // حط دي >> مكان دي <<
 // كتب /n بدل \n
-  return 0;
+  return 0; // 1/1
 // return0 مش return 1
-}
\ No newline at end of file
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit
+1 bonus mark for your comments